en/includes/guides/authentication/passwordless-login/add-passwordless-login-with-passkey.md

```diff
@@ -151,3 +152,80 @@ Follow the steps below to use an enrolled passkey to sign in to an application.
 
 !!! note
     During passkey progressive enrollment, if a user wishes to use a federated authenticator, they should have their external accounts already provisioned within {{product_name}}. If, for example, a user logs in with Google using an account not provisioned in {{product_name}}, passkey enrolment results in an error and the login flow fails.
+
+
+## Make application a FIDO trusted app
+
+If you are working with a mobile application that implements passkeys, making it a FIDO trusted app allows {{product_name}} to validate the authenticity of the application by calling the validation services of the respective platform (Android or iOS). This validation ensures that the authentication requests originate from a legitimate application, safeguarding against malicious attempts to steal credentials.
+
+!!! note 
+
+    If you wish to implement passkeys with [app-native authentication]({{base_path}}/guides/authentication/app-native-authentication/), it is mandatory to make your mobile application a FIDO trusted app. 
+
+To make the application a FIDO trusted app,
+
+1. On the {{product_name}} Console, go to **Applications** and select your application.
+
+2. In its **Advanced** tab, under **Trusted App Settings**, select **Add as a FIDO trusted app**.
+
+3. Under **Platform Settings**, enter the following platform-specific details.
+    
+    - For an Android app:
+
+        - Provide the package name of the application which takes the reverse domain format (e.g. com.example.myapp)
+
+        - Provide key hashes, which are SHA256 fingerprints of the app's signing certificate.
+    
+    - For an iOS app:
+
+        - Provide the app ID of your application which consists of the team ID and the bundle ID separated by a period (.). (e.g. A1B2C3D4E5.com.domainname.applicationname)
+    
+    !!! note
+
+        {% if product_name == "Asgardeo" %}
+        Validation services of Android and iOS require the application data under **Platform Settings** to be available in the following public endpoints common to all Asgardeo organizations. 
+
+        - For Android - `{{base_url}}/.well-known/assetlinks.json`
+
+        - For iOS - `{{base_url}}/.well-known/apple-app-site-association`
+
+        Alternatively, you may publish to a custom endpoint by setting up [custom domains]({{base_path}}/guides/branding/configure-custom-domains/). Make sure the data are in the format expected by the validation services of [Android](https://developer.android.com/training/app-links/verify-android-applinks#web-assoc){target="_blank"} and [iOS](https://developer.apple.com/documentation/xcode/supporting-associated-domains){target="_blank"} and are publicly hosted in the following locations.
```

Comment:
Can you replace the reference for Android with `https://developers.google.com/identity/smartlock-passwords/android/associate-apps-and-sites` instead of `https://developer.android.com/training/app-links/verify-android-applinks#web-assoc`